packages/adblocker-electron-example/README.md

```diff
@@ -46,7 +46,9 @@
 Blocking ads in Electron using [@cliqz/adblocker-electron](https://github.com/ghostery/adblocker/tree/master/packages/adblocker-electron).
 
 ```sh
-$ yarn start
+$ yarn build
+$ yarn start:commonjs
```

Comment:
This reads as if you are intended to run both `yarn start:commonjs` and yarn start:esm`. While it should be either-or, I suppose?